tokio/src/task/mod.rs

```diff
@@ -246,6 +246,9 @@ cfg_rt_core! {
     mod raw;
     use self::raw::RawTask;
 
+    mod scope;
+    pub use scope::{scope, scope_with_options, ScopeOptions, ScopedJoinHandle, ScopeHandle, ScopeCancelBehavior, ScopeDropBehavior};
```

Comment:
WDYT about, instead of calling these types `ScopeOptions`, `ScopeHandle`, `ScopeCancelBehavior`, etc, we make the `scope` module public, and just refer to them as `scope::Options`, `scope::Handle`, etc...

Reply:
I'm flexible regarding those. However one concern I have with names like `Option` and `Handle` is that they are already used in some places. And if users need both of them (e.g. runtime `Handle` and scope `Handle` - which is not too unreasonable - they would either need to rename them or only import the module. But maybe that problem doesn't exist, since we expect them to import only `scope` and use the individual structs in a qualified fashion as you mentioned?

Reply:
We encourage people to import only the module in `tracing-subscriber` and I was _shocked_ as to how well that worked. For instance, we now have a `fmt::Layer` and a `layer::Layer` and I don’t find them to be confusing in the slightest.